builder/image/templates/builder

```diff
@@ -112,8 +112,10 @@ if [ ! -f Dockerfile ]; then
     BUILD_OPTS+=' -v '
     BUILD_OPTS+=$(echo $CACHE_DIR)
     BUILD_OPTS+=':/tmp/cache:rw'
-    # give non-root slugbuilder user R/W perms for docker volumes
-    chown -R 2000:2000 $TMP_DIR $CACHE_DIR
+    # give slug group ownership of TMP_DIR and CACHE_DIR. This is used by the
+    # slug builder. Contents in these folders will be owned by slug:slug.
+    chown :2000 $TMP_DIR $CACHE_DIR
+    chmod g+rwx $TMP_DIR $CACHE_DIR
```

Comment:
It's not recursive, so it only applies to the folder. [+x on a folder is different to +x on a file, and is necessary to access files inside the folder](https://wiki.archlinux.org/index.php/File_permissions_and_attributes#Folders)